src/main/java/domain/Cash.java

```diff
@@ -0,0 +1,46 @@
+package domain;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+public class Cash {
+    private final int cashAmount;
+
+    public Cash(int cashAmount) {
+        this.cashAmount = cashAmount;
+    }
+
+    public int capableLottoCount(int lottoPrice) {
```

Comment:
```suggestion
    public int divide(Cash cash) {
```

- Cash 객체에서 Lotto Domain의 의존성을 제거하기 위해 Lotto.PRICE 를 외부에서 주입하도록 두었다면, 메소드 이름이나 파라미터 네이밍에서도 제거하는 것은 어떨까요
- Cash 객체를 적극적으로 활용해보는 것은 어떨까요